portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,69 @@
+#background {
+    
+}
+
+#bar1,
+#bar2,
+#bar3 {
+  width: 35px;
+  height: 5px;
+  background-color: black;
```

Comment:
My suggestion to alphabetize CSS declarations still applies:
https://github.com/MagoDelBlocco/step/pull/5#discussion_r455838507
Note that [CSS declarations](https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax#:~:text=CSS%20declarations,-Setting%20CSS%20properties&text=A%20property%20and%20value%20pair,insensitive%20by%20default%20in%20CSS) are the things that look like `key: value` pairs. .
In general, I don't have a reference or a strong preference to the order of appearance of the top-level [CSS rulesets](https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax#CSS_rulesets), and I think it makes sense to attempt and group together related rulesets.